.vscode/iTwinUISnippets.code-snippets

```diff
@@ -0,0 +1,63 @@
+{
+  // Place your iTwinUI workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and
+  // description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope
+  // is left empty or omitted, the snippet gets applied to all languages. The prefix is what is
+  // used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
+  // $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders.
+  // Placeholders with the same ids are connected.
+  // Example:
+  // "Print to console": {
+  // 	"scope": "javascript,typescript",
+  // 	"prefix": "log",
+  // 	"body": [
+  // 		"console.log('$1');",
+  // 		"$2"
+  // 	],
+  // 	"description": "Log output to console"
+  // }
+  "Separator": {
+    "prefix": "//-",
+    "body": [
+      "// ------------------------------------------------------------------------------------------------",
+      "$1"
+    ],
+    "description": "Creates a separator comment"
+  },
+  "Component": {
+    "scope": "javascriptreact,typescriptreact",
+    "prefix": "component",
+    "body": [
+      "function ${1:Name}({ ${2:props} }) {",
+      "	return (",
+      "		${3:jsx}",
+      "	);",
+      "}"
+    ],
+    "description": "React Component boilerplate"
+  },
```

Comment:
This one is not helpful/useful. We never write components like this.

Reply:
Do we want an `export default` component instead, since the examples use that a lot, or do we just want to remove this entirely?

Reply:
I think we can remove it, it's not a lot to type anyway

Reply:
Will do.

Reply:
Removed in latest commit.